### DISCUSSION TITLE
Slow/Stutter In Character select

### DISCUSSION THREAD

**Discussion by TrashSamaUwU** [ikemen-engine/Ikemen-GO#2111] [Q&A] [answered]
the game is a little slow and stuttery when in character select it seems to run perfect both in game and in any other menu. is there anyway to fix it or stabilize it (link to video included file size was to big)

https://youtu.be/aLRDyBJUrHA

**Comment by potsmugen** [comment]
It's probably because it's trying to draw hundreds of sprites (portraits) simultaneously. A combination of Ikemen needing more optimization and your system specs.

**Reply by TrashSamaUwU** [reply]
is there any way to fix it myself its not even using that much of my systems memory or cpu i have also i do have it saved to a external drive rather than my SSD could that effect performance

- AMD Ryzen 5 4500
- 32gb ddr4
- rtx 3060 12gb vram

**Reply by Kiwi1444** [reply, marked as answer]
> is there any way to fix it myself its not even using that much of my systems memory or cpu i have also i do have it saved to a external drive rather than my SSD could that effect performance
> 
>     * AMD Ryzen 5 4500
> 
>     * 32gb ddr4
> 
>     * rtx 3060 12gb vram

Even on SSD  stuttering durning character selecting in Ikemen Go sometimes perists, while in Mugen not. I'm waiting for optimisation very much
